kroxylicious-integration-tests/src/test/java/io/kroxylicious/proxy/MetricsIT.java

```diff
@@ -152,6 +165,59 @@ void prometheusMetricsWithCommonTags(KafkaCluster cluster) {
         }
     }
 
+    @Test
+    void testKroxyliciousInboundDownstreamMessages(KafkaCluster cluster, Topic topic) {
+
+        NamedFilterDefinition namedFilterDefinition = new NamedFilterDefinitionBuilder("filter-1",
+                ProduceRequestTransformation.class.getName())
+                .withConfig("transformation", TestEncoderFactory.class.getName()).build();
+
+        var config = proxy(cluster)
+                .addToFilterDefinitions(namedFilterDefinition)
+                .addToDefaultFilters(namedFilterDefinition.name())
+                .withNewManagement()
+                .withNewEndpoints()
+                .withNewPrometheus()
+                .endPrometheus()
+                .endEndpoints()
+                .endManagement();
+
+
+        try (var tester = kroxyliciousTester(config);
+                var ahc = tester.getManagementClient();
+                var producer = tester.producer(Map.of(CLIENT_ID_CONFIG, "shouldModifyProduceMessage", DELIVERY_TIMEOUT_MS_CONFIG, 3_600_000));
+                var consumer = tester
```

Comment:
I'm not sure why you've created a consumer.  This test is focused on the inbound downstream metrics, so we don't need a consumer at all.